src/components/Security/Users/Steps/Basic.vue

```diff
@@ -45,7 +46,8 @@ export default {
   },
   data() {
     return {
-      autoGenerateKuidValue: false
+      autoGenerateKuidValue: false,
+      profilesValid: null
```

Comment:
Nitpicking
```suggestion
      areProfilesValid: null
```
it's easier to know if it's a boolean